tests/gdx-tests/src/com/badlogic/gdx/tests/extensions/ControllersTest.java

```diff
@@ -1,184 +1,283 @@
-/*******************************************************************************
```

Comment:
@GreenLightning 
I've spent countless hours trying to fix this, really. Unfortunately nothing worked in my case.
So, if you find out a working solution, please, let me know.